### PR TITLE
Fix some console error and Update css design

### PR DESCRIPTION
**Pull request by nishi-ai**
CSS Hover Animation on text | Fix Form Input and add Loading on submit |
|--|--|
|<video src='https://user-images.githubusercontent.com/60405643/220446386-d66afced-8c71-408d-966d-865f0873d02c.mov' height=100 width=100></video>|<video src='https://user-images.githubusercontent.com/60405643/221142652-672a9a57-549e-4565-82f4-096209ed6b1d.mov' height=100 width=100>|

### CODE REVIEW

**Comment by benevbright** [comment]
I just tested in preview, though. But I got the error page on registration.

File: components/AddPlayerForm.js [hunk 1]
```diff
@@ -53,58 +54,72 @@ function AddPlayerForm(props) {
...
-      >
-        <span>Register</span>
-      </button>
-    </form>
+      ) : (
+        <form
+          noValidate
+          onSubmit={submitHandler}
+          validated={
+            enteredNameIsValid.toString() && enteredEmailIsValid.toString()
```
Comment (benevbright) [inline components/AddPlayerForm.js:71]: Maybe this is wrong? In preview, email validation is working weirdly. I typed until `ddd@` and the validation seems to say ok.

**Review by benevbright — COMMENTED** [review]
Looks pretty good!! 👍 I just left a comment about registration.

**Comment by benevbright** [comment]
@nishi-ai is it ready for review?

File: components/Tabs/Tabs.js [hunk 2]
```diff
@@ -47,10 +44,10 @@ const Tabs = (props) => {
           <TabPanel value={value} index={index} key={index}>
             {props.data
               .filter((item) => new Date(item.date).getFullYear() === year)
-              .map((item) => {
+              .map((item, index) => {
                 return (
                   <EventItem
-                    key={item.id}
+                    key={index}
```
Comment (benevbright) [inline components/Tabs/Tabs.js:50]: actually using `id` is recommended than using `index.

**Review by benevbright — COMMENTED** [review]
Looks very good! The error messages are placed very nicely :) cool. 

One thing is that I type `ddd@ddd.` and there is no error but validation in server fails. Maybe we can have a better validation in front end too?

File: components/Tabs/Tabs.js (hunk 2 above)
Comment (nishi-ai) [reply components/Tabs/Tabs.js:50]: but I get an error message saying `Each child in a list should have a unique "key" prop.` when I keep id.🧐

Comment (benevbright) [reply components/Tabs/Tabs.js:50]: hm, what's in `item.id`? sounds like it's all `undefined`. Is there some value in `item` that is good for an id?

**Review by benevbright — APPROVED** [review]
LGTM! one comment left but ready to go! 👍

**Comment by benevbright** [comment]
https://github.com/nishi-ai/softball-register-web/pull/65#discussion_r1125516125 was not resolved. but it's minor :)

**Comment by benevbright** [comment]
the text effect is really looking good 🙌

**Pull request merged**